tests/commands/touch.rs

```diff
@@ -1,15 +1,12 @@
-use nu_test_support::fs::Stub::EmptyFile;
 use nu_test_support::nu;
 use nu_test_support::playground::Playground;
 
 #[test]
-fn adds_a_file() {
-    Playground::setup("add_test_1", |dirs, sandbox| {
-        sandbox.with_files(vec![EmptyFile("i_will_be_created.txt")]);
-
+fn creates_a_file_when_it_doesnt_exist() {
+    Playground::setup("create_test_1", |dirs, _sandbox| {
         nu!(
-            cwd: dirs.root(),
-            "touch touch_test/i_will_be_created.txt"
```

Comment:
This test was broken for me. Not sure where the `touch_test` directory was being constructed.